tests/foreman/ui/test_syncplan.py

```diff
@@ -176,15 +189,19 @@ def test_negative_create_2(self, name):
         @Assert: Sync Plan is not created with more than 255 chars
 
         """
-        description = 'more than 255 chars'
         with Session(self.browser) as session:
-            make_syncplan(session, org=self.org_name, name=name,
-                          description=description, submit_validate=False)
+            make_syncplan(
+                session,
+                org=self.organization.name,
+                name=name,
+                description='more than 255 chars',
```

Comment:
Whoops! I missed the old `description = '...'` variable. It makes sense that you left the test code as is.